job-roles/customer-success.md

```diff
@@ -37,6 +37,7 @@ As a customer success engineer, you will be responsible for users and customers
 
 ### Customer Success Engineer
 
+* Expert knowledge on Linux OS(Centos, Ubuntu, Debian, RedHat) with at least 1-2 years experience in system administration and support role.
 * Customer Success Engineers have gained experience in installing the OpenEBS product and connecting them to the Director and in using Litmus.
```

Comment:
n_: Probably change this to "experience in installing the OpenEBS and other products from MayaData"

Reply:
updated